src/panels/lovelace/editor/unused-entities/hui-select-row.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  css,
+  CSSResult,
+} from "lit-element";
+
+import { Checkbox } from "@material/mwc-checkbox";
+import { HomeAssistant } from "../../../../types";
+import computeStateName from "../../../../common/entity/compute_state_name";
+import computeDomain from "../../../../common/entity/compute_domain";
+
+import "../../../../components/ha-checkbox";
+import "../../../../components/entity/state-badge";
+import "../../../../components/ha-relative-time";
+import "../../../../components/ha-icon";
+
+import { fireEvent } from "../../../../common/dom/fire_event";
+
+declare global {
+  // for fire event
+  interface HASSDomEvents {
+    "entity-selection-changed": EntitySelectionChangedEvent;
+  }
+}
+
+export interface EntitySelectionChangedEvent {
+  entity: string;
+  selected: boolean;
+}
+
+@customElement("hui-select-row")
+class HuiSelectRow extends LitElement {
+  @property() public hass?: HomeAssistant;
```

Comment:
Let's make it a required prop. Makes for more readable code as we don't have to check for existence or sprinkle `!` through our code.

```suggestion
  @property() public hass!: HomeAssistant;
```